packages/tools/fetch-tool/src/fluidFetchSnapshot.ts

```diff
@@ -76,8 +77,8 @@ function fetchBlobs(prefix: string,
                 reused = false;
                 blob = blobCache.get(blobId);
```

Comment:
another example but blob is Promise<string>

Reply:
Well, once you remove read() from IDocumentStorageService, you will be forces to use readBlob here, and start working with buffers, not strings. This will force downstream changes for all this code to work with buffers.
Feel free to stage such changes (i.e. postpone them till next PR by opening issues).
That said, it might be hard to debug failures where we mismatched strings encoding (base64 vs decoded string) and I feel like the easiest way to ensure we are doing everything right is to remove read() and most of base64 encoding / decoding from runtime layer, and code inspecting telling us that we mostly do not convert anywhere (other then couple exceptions, and mostly covert from IBlob to buffer). That way we will have more certainty that we did everything correctly, and it will be easier to debug any mismatches.